forc-pkg/Cargo.toml

```diff
@@ -41,9 +41,14 @@ tracing = "0.1"
 url = { version = "2.2", features = ["serde"] }
 vec1 = "1.8.0"
 walkdir = "2"
+fuel-asm = { version = "0.57.0", features = ["serde"] }
+sway-ir = { version = "0.63.6", path = "../sway-ir" }
```

Comment:
Should these dependencies only be added for the profile feature?

Reply:
Thanks for pointing these out. The`fuel-asm` crate is no longer being used and has been removed. The `sway_ir` crate is indeed only being used for the profiling functionality, so we have made this an optional dependency that only loads when using the `profiler` feature.